build/messages/src/Messages.json

```diff
@@ -47,25 +47,37 @@
     "defaultMessage": "Combines the flexible and powerful infrastructure management capabilities of Red Hat Satellite with the ability to execute remediation plans from Red Hat Insights",
     "description": "Combines the flexible and powerful infrastructure management capabilities of Red Hat Satellite with the ability to execute remediation plans from Red Hat Insights"
   },
-  "configureBasicAuthStep1": {
-    "defaultMessage": "Modify the auto_config= value to FALSE",
-    "description": "Modify the auto_config= value to FALSE"
-  },
-  "configureBasicAuthStep2": {
-    "defaultMessage": "Add your Red Hat SSO username to username",
-    "description": "Add your Red Hat SSO username to username"
-  },
-  "configureBasicAuthStep3": {
-    "defaultMessage": "Add your Red Hat SSO password to password",
-    "description": "Add your Red Hat SSO password to password"
-  },
-  "configureBasicAuthStep4": {
-    "defaultMessage": "Save the configuration",
-    "description": "Save the configuration"
-  },
-  "configureBasicAuthTitle": {
-    "defaultMessage": "Configure basic authentication in /etc/insights-client/insights-client.conf",
-    "description": "Configure basic authentication in insights-client.conf"
+  "configureActivationKeysStep1": {
+    "defaultMessage": "Generate or obtain an activation key ID on the {hybridCloudConsole}",
+    "description": "Generate or obtain an activation key ID on the {hybridCloudConsole}"
+  },
+  "configureActivationKeysStep2": {
+    "defaultMessage": "Run the following command on the host, replacing and with your information",
```

Comment:
This is missing "replacing <activation_key_name> and <organization_ID> with your information". Just checking with mocks